Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:9-slim
+WORKDIR /app
+COPY package.json /app
+RUN npm install
```

Comment:
Just realized this when testing this.... the project is using `yarn` and not `npm`

Reply:
though it is still fine as this will be inside a container